### PR TITLE
Converted paper-dialog to ha-paper-dialog

### PR DESCRIPTION
**Pull request by yosilevy**
Maybe ha-dialog should be placed in top level components folder. But that's going to be a lot of work...

### CODE REVIEW

File: hassio/src/hassio-markdown-dialog.js [hunk 1]
```diff
@@ -1,18 +1,18 @@
 import "@polymer/app-layout/app-toolbar/app-toolbar";
 import "@polymer/paper-dialog-scrollable/paper-dialog-scrollable";
-import "@polymer/paper-dialog/paper-dialog";
 import "@polymer/paper-icon-button/paper-icon-button";
 import { html } from "@polymer/polymer/lib/utils/html-tag";
 import { PolymerElement } from "@polymer/polymer/polymer-element";
 
 import "../../src/components/ha-markdown";
 import "../../src/resources/ha-style";
+import "../../src/panels/lovelace/components/dialog/ha-dialog";
```
Comment (balloob) [inline hassio/src/hassio-markdown-dialog.js]: this shouldn't be part of the lovelace components but instead be part of `src/components` and name should match the component: `ha-paper-dialog`.

File: src/panels/lovelace/editor/card-editor/hui-edit-card.ts [hunk 2]
```diff
@@ -428,7 +428,7 @@ export class HuiEditCard extends LitElement {
         }
 
         @media (min-width: 1200px) {
-          paper-dialog {
+          ha-paper-dialog {
```
Comment (balloob) [inline src/panels/lovelace/editor/card-editor/hui-edit-card.ts]: This seems weird. If there is no paper dialog on this page, we can remove this CSS Declaration ?

File: hassio/src/hassio-markdown-dialog.js (hunk 1 above)
Comment (yosilevy) [reply hassio/src/hassio-markdown-dialog.js]: See my comment at the top. Knew you were going to say that...

File: src/panels/lovelace/editor/card-editor/hui-edit-card.ts (hunk 2 above)
Comment (yosilevy) [reply src/panels/lovelace/editor/card-editor/hui-edit-card.ts]: But there is. https://github.com/home-assistant/home-assistant-polymer/blob/a9bd53432e84a05a4933a09470468f4e4efcd6fc/src/panels/lovelace/editor/card-editor/hui-edit-card.ts#L151

File: hassio/src/hassio-markdown-dialog.js (hunk 1 above)
Comment (yosilevy) [reply hassio/src/hassio-markdown-dialog.js]: Done

File: src/panels/lovelace/editor/card-editor/hui-edit-card.ts (hunk 2 above)
Comment (balloob) [reply src/panels/lovelace/editor/card-editor/hui-edit-card.ts]: Oh it was broken before 😛

**Review by balloob — APPROVED** [review]

**Pull request merged**